src/sql/src/kafka_util.rs

```diff
@@ -11,14 +11,18 @@
 
 use std::collections::BTreeMap;
 use std::convert;
+use std::convert::TryInto;
 use std::fs::File;
 use std::io::Read;
 use std::sync::Mutex;
+use std::time::SystemTime;
 
 use anyhow::bail;
 use log::{debug, error, info, warn};
 use rdkafka::client::ClientContext;
 use rdkafka::consumer::{BaseConsumer, Consumer, ConsumerContext};
+use rdkafka::Offset;
```

Comment:
you could do something similar here, if you'd like! (`use rdkafka::{Offset, TopicPartitionList}`)

Reply:
Thanks and done! I _thought_ these were merged automatically by `rustfmt`, but I apparently was confused. 🙈

Reply:
There is an option, but we may have it disabled as it is quite assertive. 

https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=#imports_granularity